### ISSUE TITLE
Android app

### ISSUE THREAD

**Issue by FinixFighter** [pixelfed/support#149]
Hi! I just wanted to ask you if there will be a Pixelfed app for Android in the future? It would be nice to have!

**Comment by escuco** [comment]
Do you know Pixeldroid?
https://pixeldroid.org/

**Comment by FinixFighter** [comment]
> Do you know Pixeldroid? https://pixeldroid.org/

Nice! I didn't know that, thank you! That's not an official app though, I suppose.

**Comment by escuco** [comment]
Yes, that's right. The app works for me and at the moment the only way I know to get notifications.

**Comment by piratenpanda** [comment]
Official Android app is in the works according to mastodon posts :)

**Comment by eliocamp** [comment]
There are now official android and iOS app: https://pixelfed.org/mobile-apps

This issue could be closed now, I think.
